h5p-bildetema/src/components/LanguageDropdown/LanguageDropdown.tsx

```diff
@@ -28,18 +28,33 @@ export const LanguageDropdown: React.FC<LanguageDropdownProps> = ({
   search,
   topicIds,
 }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleOnClickOutside = (): void => {
+    handleSelectorVisibility(false);
+    setIsActive(false);
+  };
+
   const dropdownRef = useDetectClickOutside({
-    onTriggered: () => handleSelectorVisibility(false),
+    onTriggered: handleOnClickOutside,
   });
 
+  const handleOnClick = (): void => {
```

Comment:
Would it make sense to make the button inactive when a language is selected and the dropdown closes?

Reply:
Yes!